agent/pairwise/pairwise.go

```diff
@@ -31,7 +31,7 @@ type Callee struct {
 
 func (p *Callee) startStore() {
 	wallet := p.agent.Wallet()
-	p.Caller.Store(wallet)
+	p.Caller.Store(p.agent.ManagedWallet())
```

Comment:
as also we talked about, the `startStore()` function is difficult to follow when we don't have only one logical wallet or similar == future topics.